questions/qa-bidi-unicode-controls.en.html

```diff
@@ -64,20 +64,25 @@ <h1>How to use Unicode controls for bidi text</h1>
     <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
   </div>
     
-<section id="question">
-<h2>Question</h2>
-  <p class="question">If I'm unable to use markup to correctly order bidirectional text, what can I do?</p>
-  <div class="insideinfonote">
-    <p class="noprint info">Right-to-left text in code samples is represented by UPPERCASE TRANSLATIONS, and left-to-right text by lowercase. </p>
-    <p>If there is a  nearby <img src="../icons/codelink.gif" alt="View code."> image, click on it to see  how the example looks in your browser. From the page that opens you can  view the source code for the example.</p>
-  </div>
-<p>This article assumes that you are familiar with bidirectional text concepts and managing bidirectional text using HTML markup, but that you need to know how to do similar things with Unicode control characters, such as when writing plain text.  If you are not familiar with bidi in HTML, you may find it helpful to first read through the article <a href="/International/articles/inline-bidi-markup/">Inline markup and bidirectional text in HTML</a>.</p>
-</section>
+
+
+<p>This article looks at how content authors can apply direction metadata to bidirectional text when markup is not available.</p>
```

Comment:
I'm struggling with whether this is metadata here or not? Perhaps...

```suggestion
<p>This article shows how content authors can overcome common directional problems with bidirectional text when markup is not available.</p>
```

Reply:
It's not information that the user reads as part of the text (in fact they are invisible anyway), but it provides information about the text.  I think  that qualifies as metadata.  If it were in HTML, we'd be talking about the dir attribute values - for me that's metadata too.